internal/service/alarms/serve.go

```diff
@@ -32,7 +34,28 @@ const (
 
 // Serve TODO: Call this func using cobra-cli from inside deployment CR.
 func Serve() {
-	// TODO: Init client-go
+	slog.Info("Starting Alarm server")
+
+	// Channel for shutdown signals
+	shutdown := make(chan os.Signal, 1)
+	signal.Notify(shutdown, os.Interrupt, syscall.SIGTERM)
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	go func() {
+		<-shutdown
+		cancel()
+	}()
+
+	// Get client for hub
+	hubClient, err := clients.NewClientForHub()
+	if err != nil {
+		slog.Error("error creating client for hub", "error", err)
+		os.Exit(1)
+	}
+
+	alarmsDict := dictionary.New(hubClient)
+	alarmsDict.Load(ctx)
```

Comment:
Is there any clean up we need to do after this...like closing the client maybe?

Reply:
Good question. I wondered the same, especially for the managed cluster clients. No, we don't have to close anything.